docs/guide/configuration.md

```diff
@@ -1,119 +1,63 @@
 # Configuration
 
-## Content Repository configuration
+eZ Platform configuration is delivered using a number of dedicated configuration files.
+This config covers everything from selecting the content Repository to SiteAccesses to language settings.
 
-The default storage engine for the Repository is called Legacy storage engine.
+### Configuration format
 
-You can define several Repositories within a single application. However, you can only use one per site.
+The recommended configuration format is YAML. It is used by default in the kernel (and in examples throughout the documentation).
+However, you can also use XML or PHP formats for configuration.
 
-### Configuration examples
+Basic configuration handling in eZ Platform is similar to the usual Symfony config. To use it, you define key/value pairs in your configuration files. Internally and by convention, keys follow a dot syntax where the different segments follow your configuration hierarchy. Keys are usually prefixed by a namespace corresponding to your application. All kinds of values are accepted, including arrays and deep hashes.
 
-#### Using default values
+### Configuration files
 
-``` yaml
-# ezplatform.yml
-ezpublish:
-    repositories:
-        # Defining Repository with alias "main"
-        # Default storage engine is used, with default connection
-        # Equals to:
-        # main: { storage: { engine: legacy, connection: <defaultConnectionName> } }
-        main: ~
+Main configuration files are located in the `app/config` folder.
 
-    system:
-        # All members of my_siteaccess_group will use "main" Repository
-        # No need to set "repository", it will take the first defined Repository by default
-        my_siteaccess_group:
-            # ...
-```
-
-If no Repository is specified for a SiteAccess or SiteAccess group, the first Repository defined under `ezpublish.repositories` will be used.
+- `parameters.yml` contains infrastructure-related configuration. It is created based on the default settings defined in `parameters.yml.dist`.
+- `config.yml` contains configuration stemming from Symfony and covers settings such as search engine or cache configuration.
+- `ezplatform.yml` contains general configuration that is specific for eZ Platform, like for example SiteAccess settings.
+- `security.yml` is the place for security-related settings.
+- `routing.yml` defines routes that will be used throughout the application.
 
-#### All explicit
+Configuration can be made environment-specific using separate files for each environment. These files will contain additional settings and point to the general (not environment-specific) configuration that will be applied in other cases.
```

Comment:
1. These files should contain / These files contain
2. configuration that is applied in other cases.